src/app/api/api_v1/endpoints/cameras.py

```diff
@@ -66,10 +76,9 @@ async def create_camera_token(
     token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN]),
 ) -> Token:
     telemetry_client.capture(token_payload.sub, event="cameras-token", properties={"camera_id": camera_id})
-    # Verify camera
     await cameras.get(camera_id, strict=True)
     # create access token using user user_id/user_scopes
-    token_data = {"sub": str(camera_id), "scopes": ["camera"]}
+    token_data = {"sub": str(camera_id), "scopes": ["camera"], "organization_id": token_payload.organization_id}
```

Comment:
If I'm not mistaking, in the JWT payload, we want the org ID of the camera, not the authenticated admin

Reply:
ah yes sorry I fixed that in a following PR by mistake